kubernetes/e2e_test/test_k8sclient.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
```

Comment:
would you please add copyright header to kubernetes/e2e_test/__init__.py file too. annoying but I think it is required on all files.